.github/workflows/tests.yml

```diff
@@ -116,6 +121,7 @@ jobs:
           python -m pip install --upgrade pip
           pip install -r requirements-ci.txt
           pip install wheel pytest pytest-cov pytest-mock flaky --upgrade
+          pip3 install openfermionpyscf

```

Comment:
@soranjh how come the device tests need to install openfermion?

Reply:
They do not run all tests including the qchem ones? I can remove installing openfermionpyscf here, then all tests needing openfermionpyscf will get skipped.

Reply:
No, the device tests simply run `pennylane/devices/tests` against a variety of devices. All these tests do is execute various QNodes, and ensure that the device returns the correct result :)

Reply:
Thanks! Removed it.